logstash-core/lib/logstash/config/pipelines_info.rb

```diff
@@ -93,7 +91,7 @@ def self.format_pipeline_vertex_section_stats(stats, pipeline)
 
             nested_type = if type_sym == :"counter/long"
                             :long_counters
-                          elsif type_sym == :"gauge/numeric"
+                          elsif type_sym == :"gauge/number"
```

Comment:
See the metrics definition: https://github.com/elastic/logstash/blob/main/logstash-core/src/main/java/org/logstash/instrument/metrics/MetricType.java#L57